tests/markers.py

```diff
@@ -0,0 +1,7 @@
+"""Shared pytest functionality."""
+import platform
+import pytest
+
+
+windows_only = pytest.mark.skipif(platform.system() != "Windows", reason="Windows required")
```

Comment:
Could we use the utilities from unittest instead? i.e. @unittest.skipIf https://docs.python.org/3/library/unittest.html#skipping-tests-and-expected-failures.